apps/dashboard/src/components/analytics/auto-bar-chart.tsx

```diff
@@ -130,126 +127,121 @@ export const AutoBarChart = <
   }
 
   return (
-    <Box {...boxProps}>
-      <ResponsiveContainer width="100%" height="100%">
-        <RechartsBarChart data={data}>
-          <defs>
-            {categories.map((cat) => (
-              <linearGradient
-                key={`${cat.id as string}`}
-                id={`bar_color_${id}_${cat.id as string}`}
-                x1="0"
-                y1="0"
-                x2="0"
-                y2="100%"
-                gradientUnits="userSpaceOnUse"
-              >
-                <stop
-                  offset="0"
-                  stopColor={cat.color || "#3385FF"}
-                  stopOpacity={1}
-                />
-                <stop
-                  offset="1"
-                  stopColor={cat.color || "#3385FF"}
-                  stopOpacity={0.75}
-                />
-              </linearGradient>
-            ))}
-          </defs>
-
+    <ResponsiveContainer width="100%" height="100%">
```

Comment:
this file is also (still?) used in the contract analytics page, just confirming it still works as expected there, too?

Reply:
Yeah - works as usual